tests/python/shared/assets/users.json

```diff
@@ -62,7 +62,7 @@
       "is_active": true,
       "is_staff": true,
       "is_superuser": true,
-      "last_login": "2022-03-05T09:37:01.636000Z",
+      "last_login": "2024-03-21T20:49:58.088000Z",
```

Comment:
```suggestion
    "last_login": "2022-03-05T09:37:01.636000Z",
```

Reply:
Yes, I understand, but it's not really needed to be updated. Changing it only introduces extra merge conflicts later.

Reply:
Hello, just a small follow-up on this, if these particular changes aren't needed. So, shouldn't we consider changing the dumping JSON code so as not to dump such values? Each time we look through a dump, it might be tiring to revert the unnecessary parts.

Reply:
This goes for both `data.json` and `user.json`

Reply:
I think, it can be added with as a default option with a parameter to dump everything.

Reply:
Do you mean to say, like a flag `-i` to indicate only ones needed in the test to be updated, like what we wanted here? Or else dump everything.

Reply:
Ah, that makes sense. Should I open an issue related to this?

Reply:
It doesn't seem like a real issue to me, but you can create it, if you see it's a problem.